dev-support/checks/unit.sh

```diff
@@ -65,6 +65,12 @@ for i in $(seq 1 ${ITERATIONS}); do
   fi
 
   if [[ ${ITERATIONS} -gt 1 ]]; then
+    if ! grep -q "Running .*Test" "${REPORT_DIR}/output.log"; then
```

Comment:
The pattern `"Running .*Test"` assume that the test name contains `Test`; e.g.
```
$grep "Running .*Test" ~/Downloads/result-3-9475609402-split-16/iteration4/output.log 
[INFO] Running org.apache.hadoop.fs.ozone.TestHSync
```
It may be better to grep `"Tests run: [1-9]"`.
```
$grep "Tests run: [1-9]" ~/Downloads/result-3-9475609402-split-16/iteration4/output.log 
[ERROR] Tests run: 10, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 154.095 s <<< FAILURE! - in org.apache.hadoop.fs.ozone.TestHSync
[ERROR] Tests run: 10, Failures: 0, Errors: 1, Skipped: 0
```
Then, it will work for any test names.

Reply:
Thanks @szetszwo for the review.

`Tests run:` is not printed if there is a fork timeout (Ozone used to have a similar pattern before HDDS-10545).

```
[INFO] Running org.apache.hadoop.ozone.client.rpc.TestECKeyOutputStreamWithZeroCopy
[INFO] 
[INFO] Results:
...
... There was a timeout or other error in the fork
```

While this workflow allows any test to be run, Ratis limits test naming by:

https://github.com/apache/ratis/blob/dac27e4fd3df042fcf50ed549404e4d82708ed2b/pom.xml#L647-L649

Reply:
I see.  Thanks for the info.